docs/guide/previews.md

```diff
@@ -6,11 +6,7 @@ parent: Guide
 
 # Previews
 
-`ViewComponent::Preview`, like `ActionMailer::Preview`, provides a quick way to preview components in isolation.
-
-_For a more interactive experience, consider using [ViewComponent::Storybook](https://github.com/jonspalmer/view_component_storybook) or [Lookbook](https://github.com/allmarkedup/lookbook)._
```

Comment:
Maybe keep the mention of these 2 tools, but move it to the end of the page?

Reply:
Agreed! I re-added this 👍🏻